mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -90,6 +108,12 @@ private[regression] trait AFTSurvivalRegressionParams extends Params
       SchemaUtils.checkColumnType(schema, $(censorCol), DoubleType)
       SchemaUtils.checkColumnType(schema, $(labelCol), DoubleType)
     }
+    if (hasQuantileProbabilities && hasQuantilesCol) {
+      SchemaUtils.appendColumn(schema, $(quantilesCol), new VectorUDT)
+    } else if (hasQuantileProbabilities || hasQuantilesCol) {
+      logWarning("The output will not include quantiles column. " +
+        "If you want to include, please set both quantileProbabilities and quantilesCol.")
+    }
```

Comment:
I think if users set `quantilesCol` but not `quantileProbabilities`, we should throw an exception rather than an warning message because users should expect `quantilesCol` in the output. Actually, I think it is more convenient to set the default value of `quantileProbabilities` to `[0.25, 0.5, 0.75]`, and then require a non-empty array for this param.